sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -205,7 +248,8 @@ def test_spectral_embedding_callable_affinity(X, seed=36):
 @pytest.mark.filterwarnings(
     "ignore:scipy.linalg.pinv2 is deprecated:DeprecationWarning:pyamg.*"
 )
-def test_spectral_embedding_amg_solver(seed=36):
+@pytest.mark.parametrize("dtype", (np.float32, np.float64))
```

Comment:
since we have our created our marker, let's use it here.

```suggestion
@pytest.mark.skipif(
    is_pyamg_not_available, reason="PyAMG is not installed and thus we cannot test."
)
@pytest.mark.parametrize("dtype", (np.float32, np.float64))
```